GWAS/PCA.ipynb

```diff
@@ -522,12 +617,14 @@
    "outputs": [],
    "source": [
     "# Plot PCA results. Can be used:\n",
+    "# independently as \"plot_pca\" or conbinded with other workflow as \"project_samples+plot_pca\"\n",
     "[plot_pca]\n",
     "parameter: outlier_file = path()\n",
     "parameter: plot_data = path\n",
+    "parameter: plot_K = int\n",
     "input: plot_data\n",
-    "output: f'{cwd}/{_input:bn}.pc.png',\n",
```

Comment:
Again, for when input data may come from readonly folder, let's write the output to cwd -- please revert the changes to that.